### PR TITLE
style: tweak hp comparison styles

### PR DESCRIPTION
**Pull request by hkang1**
## Description

- Fix the issue of aligning the arrow within an `antd SelectFilter`
- Add overflow scrolling to table body to avoid having the modal scroll the entire page (also to preserve trial header)
- Minor style tweaks.

BEFORE:
Screen Shot 2021-08-13 at 9 55 45 AM

AFTER:
Screen Shot 2021-08-13 at 9 55 35 AM

## Checklist

- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
See [Release Note](../docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234

### CODE REVIEW

File: webui/react/src/styles/antd.scss [hunk 1]
```diff
@@ -92,11 +92,11 @@ html {
...
 
   .ant-select > .ant-select-arrow {
-    transform: rotate(0deg);
+    transform: translateY(-50%) rotate(0deg);
     transition: transform 0.2s linear;
   }
   .ant-select.ant-select-open > .ant-select-arrow {
-    transform: rotate(180deg);
+    transform: translateY(-50%) rotate(180deg);
   }
```
Comment (EmilyBonar) [inline webui/react/src/styles/antd.scss:100]: This breaks the arrow in other places
Screenshot 2021-08-13 at 11-07-49 Experiment 77 - Determined

**Review by EmilyBonar — APPROVED** [review]
Looks good, just that one issue with the arrow

File: webui/react/src/styles/antd.scss (hunk 1 above)
Comment (hkang1) [reply webui/react/src/styles/antd.scss]: weird, I rebased with your changes, and then I took out the arrow changes and it looks fine now...

Comment (EmilyBonar) [reply webui/react/src/styles/antd.scss]: Weird, it still looks bad on latest-master, but is fine on netlify

**Pull request merged**